src/main/java/File/FileManager.java

```diff
@@ -0,0 +1,222 @@
+            String last = "";
+            for (int i = 0; i < l; i++) {
+                last = last + current.charAt(i);
+            }
+            date = date + last;
+        } catch (NoSuchElementException e) {
+            System.out.println(e);
+        }
+        DateTimeFormatter d = DateTimeFormatter.ofPattern("dd MMM yyyy HHmm");
+        LocalDateTime dt = LocalDateTime.parse(date, d);
+        return dt.format(DateTimeFormatter.ofPattern("dd-MM-uuuu HHmm"));
+    }
+
+    /**
+     * Returns the type of task in the file.
+     *
+     * @param s text line.
+     * @return int type of task.
+     * @throws ErrorExceptions error reading the type in the file.
+     */
+    private static int getType(String s) throws ErrorExceptions {
+        Scanner sc = new Scanner(s);
+        try {
+            String current = sc.next();
+            char type = current.charAt(1);
+            if (type == 'T') {
+                return 1;
+            } else if (type == 'D') {
+                return 2;
+            } else if (type == 'E') {
+                return 3;
+            } else {
+                throw new ErrorExceptions("Error: Wrong item type detected, file might be corrupted!");
+            }
+        } catch (NoSuchElementException e) {
+            throw new ErrorExceptions("Failed to load saved file (Type)" + System.lineSeparator()
+                    + e);
+        }
+    }
+
+    /**
+     * Returns whether the task is completed.
+     *
+     * @param s text line.
+     * @return boolean completed or not completed.
+     * @throws ErrorExceptions error when reading the file.
+     */
+    private static boolean getDone(String s) throws ErrorExceptions {
+        Scanner sc = new Scanner(s);
+        try {
+            String current = sc.next();
+            char done = current.charAt(4);
+            String d = "";
+            d = d + done;
+            if (d.equals("O")) {
+                return true;
+            } else if (d.equals("X")) {
+                return false;
+            } else {
+                throw new ErrorExceptions("Error: Cannot determine if item is done, file might be corrupted!");
+            }
+        } catch (NoSuchElementException e) {
+            throw new ErrorExceptions("Failed to load saved file (Completed?)" + System.lineSeparator()
+                    + e);
+        }
+    }
+
+    /**
+     * Reads the local save file and convert into task to add to list.
+     *
+     * @param f saved file.
+     * @param store ArrayList to store the tasks.
+     */
+    public static void read(File f, ArrayList<task> store) {
+        try {
+            Scanner sc = new Scanner(f);
+            while (sc.hasNext()) {
+                String current = sc.nextLine();
+                try {
+                    int type = FileManager.getType(current);
+                    String name = FileManager.getName(current);
+                    boolean done = FileManager.getDone(current);
```

Comment:
Hello, according to the coding standards, `boolean` values should be named to sound like booleans. An alternative here could be `isDone`